3rdParty/fuerte/src/H1Connection.h

```diff
@@ -89,6 +91,16 @@ class H1Connection final : public fuerte::GeneralConnection<ST, RequestItem> {
 
   virtual std::unique_ptr<RequestItem> createRequest(
       std::unique_ptr<Request>&& req, RequestCallback&& cb) override {
+#ifdef ARANGODB_ENABLE_FAILURE_TESTS
+    if (req->getFuzzerReq()) {
+      return std::make_unique<RequestItem>(
+          std::move(req), std::move(cb), req->getFuzzReqHeader().value_or(""));
```

Comment:
the value_or("") is probably unnecessary here.

Reply:
Addressed